tools/apigee/apigee.py

```diff
@@ -21,13 +21,17 @@
 from google.auth import default
 from google.auth.transport.requests import Request
 from httpx import AsyncClient
+from httpx import HTTPStatusError
 from httpx import Limits
 
 _BILLING_PROJECT_ID = "datcom-204919"
 _API_KEYS_BASE_URL = "https://apikeys.googleapis.com"
+_APIGEE_BASE_URL = "https://apigee.googleapis.com"
 _HTTPX_LIMITS = Limits(max_keepalive_connections=5, max_connections=10)
+_HTTP_RESOURCE_EXISTS_CODE = 409
 
 _DC_API_TARGET = os.environ.get("DC_API_TARGET", "api.datacommons.org")
+_APIGEE_ORGANIZATION = os.environ.get("APIGEE_ORGANIZATION", "datcom-apigee")
```

Comment:
I think i'd be safer to default to datcom-apigee-dev here

Reply:
Updated so that the variables are required now and have no defaults.